conntrack/conntrack.go

```diff
@@ -88,7 +89,11 @@ func (c Conntrack) RemoveConntrackFlows(ipVersion uint8, ipAddr net.IP) {
 			if err == nil {
 				logCxt.Debug("Successfully removed conntrack flows.")
 				break
+			} else if bytes.Contains(output, []byte("0 flow entries have been deleted")) {
+				logCxt.Debug("conntrack tool didn't find any flows.")
+				break
```

Comment:
How is this addition helping?

Reply:
Added a comment; conntrack treats it as a failure if it doesn't find any flows to delete (but for us that's a success!)

Reply:
Yes, I figured that out later :facepalm: